src/docs/development/tools/devtools/code-size.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Using the Code Size Tool
+description: Learn how to use the DevTools code size tool.
+---
+
+## What is it?
+
+The code size tool allows you to visualize, analyze, and diff code / app size
+data.
+
+## How to use it
+
+If DevTools is already connected to a running application, navigate to the
+"Code Size" tab.
+
+![Screenshot of code size tab]({% asset tools/devtools/code_size_tab.png @path %})
+
+If DevTools is not connected to a running application, you can
+access the tool from the landing page that appears once you have launched
+DevTools.
+
+![Screenshot of code size access on landing page]({% asset tools/devtools/code_size_access_landing_page.png @path %})
+
+## Analysis Tab
+
```

Comment:
Does this need a line of text (and maybe some accessibility text)? Might be a bit out of the blue if it's just a picture under a header, and above another header :)